tools/scripts/sdk_package_json.mjs

```diff
@@ -11,6 +11,7 @@ const toAbsoluteDir = (relativeDir) => path.resolve(rootDir, relativeDir);
 
 const esmPackageJson = {
   type: 'module',
+  sideEffects: false,
```

Comment:
This is for ESM tree-shaking as a hint to bundlers https://webpack.js.org/guides/tree-shaking/#mark-the-file-as-side-effect-free